descartes_ikfast/test/descartes_ikfast_fanuc_m20ia10l_manipulator.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DESCARTES_IKFAST_FANUC_M20IA10L_MANIPULATOR_H
+#define DESCARTES_IKFAST_FANUC_M20IA10L_MANIPULATOR_H
+
+#include <descartes_ikfast/ikfast_kinematics.h>
+
+namespace descartes_ikfast_unit
+{
+
+template<typename FloatType>
+class FanucM20ia10lKinematics : public descartes_light::IKFastKinematics<FloatType>
+{
+public:
+  FanucM20ia10lKinematics(const Eigen::Transform<FloatType, 3, Eigen::Isometry>& world_to_robot_base,
```

Comment:
Could possibly eliminate this constructor with the `using` syntax:
``` c++
using descartes_light::IKFastKinematics<FloatType>::IKFastKinematics
```

Reply:
I originally tried this but I was getting compiler errors stating that it could not find a constructor. I could not figure out why, but when I explicitly defined it, it compiles and runs.